a8c-files.php

```diff
@@ -71,7 +71,7 @@ private function init_jetpack_photon_filters() {
 		// This results in all images being full size (which is not ideal)
 		add_filter( 'jetpack_photon_development_mode', '__return_false', 9999 );
 
-		if ( false === is_vip_go_srcset_enabled() ) {
+		if ( false === is_vip_go_srcset_enabled() || ! isset( $_GET['enable_vip_srcset'] ) || '1' !== $_GET['enable_vip_srcset'] ) {
```

Comment:
For simplicity, we should move the GET param checks to `is_vip_go_srcset_enabled`.